### DISCUSSION TITLE
If anyone has an idea please share a proper Video of Google authentication with Cypress

### DISCUSSION THREAD

**Discussion by shitalmeghanathi** [cypress-io/cypress#21322] [General]
### What would you like?

If anyone has an idea please share a proper Video of Google authentication with Cypress

**Comment by muratkeremozcan** [comment]
No vidoes, or working repos on my end. But, there are plenty of resources.

https://github.com/lirantal/cypress-social-logins

https://filiphric.com/google-sign-in-with-cypress

https://lzomedia.com/blog/testing-react-google-login-with-cypress/

https://docs.cypress.io/guides/testing-strategies/google-authentication#What-you-ll-learn
